dashboard/app/models/peer_review.rb

```diff
@@ -145,7 +145,19 @@ def self.get_peer_review_summaries(user, script)
         script.professional_learning_course? &&
         Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
 
-      PeerReview.where(reviewer: user, script: script).map(&:summarize)
+      PeerReview.where(reviewer: user, script: script).map(&:summarize).tap do |reviews|
+        if script.peer_reviews_to_complete &&
+            reviews.size < script.peer_reviews_to_complete &&
+            PeerReview.get_review_for_user(script, user)
+          reviews << {
+              status: 'not_started',
+              name: I18n.t('peer_review.review_new_submission'),
+              result: ActivityConstants::UNSUBMITTED_RESULT,
+              icon: '',
+              locked: false
+          }
```

Comment:
I'm a bit unclear what this is doing. It adds a new `not_started` entry to the returned list if the # of existing reviews < # required for the script and a review exists that can be reviewed? Is `PeerReview.get_review_for_user` called here to see if one is available? But this doesn't persist anything. It just shows the "Review a new submission" option?